### DISCUSSION TITLE
Provide documentation on AuthType

### DISCUSSION THREAD

**Discussion by JessHolle** [OpenIDC/mod_auth_openidc#1003] [Ideas]
There seems to be no real documentation on this setting.

I only found the "auth-openidc" value that I needed by searching the source code.  It is also mentioned in a one line change log entry from ~5 years ago.  That seems like really insufficient documentation for such an important feature/setting.

**Comment by zandbelt** [comment]
see https://github.com/zmartzone/mod_auth_openidc/wiki/Single-Page-Applications#allowing-both-oauth-20-and-openid-connect

**Reply by JessHolle** [reply]
Ah, thanks.

I didn't search the Wiki, which is my mistake I guess.

I did search the core documentation that's actually part of the Git repo, which I'd have expected to include all core reference documentation (or links to it).
